### PR TITLE
Keccak input registers

### PR DESCRIPTION
**Pull request by wborgeaud**
Add input registers to Keccak for wiring with Cpu table.

### CODE REVIEW

File: evm/src/keccak/keccak_stark.rs [hunk 1]
```diff
@@ -565,16 +608,10 @@ mod tests {
...
         let base = F::from_canonical_u64(1 << 32);
-        for x in 0..5 {
-            for y in 0..5 {
-                output.push(
-                    last_row[reg_a_prime_prime_prime(x, y)]
-                        + base * last_row[reg_a_prime_prime_prime(x, y) + 1],
-                );
-            }
-        }
+        let output = (0..INPUT_LIMBS)
```
Comment (npwardberkeley) [inline evm/src/keccak/keccak_stark.rs:612]: Should we change `INPUT_LIMBS` --> `NUM_LIMBS`? Since it's equally the number of outputs

**Review by npwardberkeley — APPROVED** [review]
Looks good1

File: evm/src/keccak/keccak_stark.rs (hunk 1 above)
Comment (wborgeaud) [reply evm/src/keccak/keccak_stark.rs:612]: What do you think about `NUM_INPUTS`, to avoid confusion since there are `2*INPUT_LIMBS` 32-bit limbs?

**Comment by npwardberkeley** [comment]
Ah yes, that makes sense!

**Pull request merged**